inventory/byo/hosts.ose.example

```diff
@@ -363,6 +363,8 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #
 # Override metricsPublicURL in the master config for cluster metrics
 # Defaults to https://hawkular-metrics.{{openshift_master_default_subdomain}}/hawkular/metrics
+# Currently, you may only alter the hostname portion of the url, alterting the
+# `/hawkular/metrics` path will break installation of metrics.
 #openshift_hosted_metrics_public_url=https://hawkular-metrics.example.com/hawkular/metrics
```

Comment:
it's actually a hostname and not url ...
maybe rename to `openshift_hosted_metrics_public_hostname` ?

checked in the console and it works :+1: 
checked in CFME and it works :+1: 

my inventory file: https://github.com/yaacov/miq-scripts/blob/master/hosts-eng-lab.local

screenshot:

![screenshot-2016-09-22-11-24-45](https://cloud.githubusercontent.com/assets/2181522/18741337/51b0a5f6-80b7-11e6-9d56-1fdb3f9f3a83.png)